src/main/java/uk/gov/hmcts/reform/managecase/service/noc/ApplyNoCDecisionService.java

```diff
@@ -247,13 +249,34 @@ private void nullifyNode(JsonNode node, String... ignoreNestedFields) {
             String errorMessage = status == HttpStatus.NOT_FOUND
                 ? String.format("Organisation with ID '%s' can not be found.", organisationId)
                 : String.format("Error encountered while retrieving organisation users for organisation ID '%s': %s",
-                organisationId, reasonPhrase);
+                                organisationId, reasonPhrase);
 
             throw new ValidationException(errorMessage, e);
         }
         return getIntersection(existingCaseAssignments, usersByOrganisation.getUsers());
     }
 
+    private Pair<List<CaseUserRole>, List<ProfessionalUser>> getUsersWithCaseAccess(
+        List<CaseUserRole> existingCaseAssignments,
+        String organisationId,
+        String caseRoleId) {
+        FindUsersByOrganisationResponse usersByOrganisation;
+        try {
+            usersByOrganisation = prdRepository.findUsersByOrganisation(organisationId);
+        } catch (FeignException e) {
+            HttpStatus status = HttpStatus.resolve(e.status());
+            String reasonPhrase = status == null ? e.getMessage() : status.getReasonPhrase();
+
+            String errorMessage = status == HttpStatus.NOT_FOUND
+                ? String.format("Organisation with ID '%s' can not be found.", organisationId)
+                : String.format("Error encountered while retrieving organisation users for organisation ID '%s': %s",
+                organisationId, reasonPhrase);
+
+            throw new ValidationException(errorMessage, e);
+        }
```

Comment:
Could move this bit into a `private` method - looks identical to in the other method?

Reply:
Good catch. Done.